scripts/EFI_workflow.R

```diff
@@ -0,0 +1,242 @@
+library("PEcAn.all")
+library("PEcAn.utils")
+library("RCurl")
+library("REddyProc")
+library("tidyverse")
+library("furrr")
+library("R.utils")
+library("dynutils")
+
+###### Preping Workflow for regular SIPNET Run ##############
+args = list()
+args$settings = "/projectnb/dietzelab/ahelgeso/Site_XMLS/los.xml"
+args$continue = TRUE
+outputPath <- "/projectnb/dietzelab/ahelgeso/Site_Outputs/Lost_Creek/"
+
+  
+if(!dir.exists(outputPath)){dir.create(outputPath, recursive = TRUE)}
+setwd(outputPath)
+
+# Open and read in settings file for PEcAn run.
+settings <- PEcAn.settings::read.settings(args$settings)
+
+start_date <- as.Date('2021-06-01')
+end_date<- as.Date('2021-06-02')
```

Comment:
So my only hesitation with that and the reason I let it hard coded, is because right now between the debugging and catching up on running forecasts now that the met data collection scripts are working it was easier to have it hard coded so I could change the date to fit the situation. 

Also we have some sites we want to run daily and other monthly so that was a another reason I left the dates as hard coded. But I think now that the met download scripts are working I want to transition the dates to args, but given the variety of dates you might want to run with this workflow I am unsure the best way to have them as args?

Reply:
1. we'd agreed that we were running everything daily
2. this script is unusable in a cron job until you remove hard-coded dates. It was fine to hard-code during debugging, but now that the code is running you need to clean stuff like this up
3. If you pass the dates in as external args, then you can run the script for any dates.